docs/generate-time-series/cli.rst

```diff
@@ -0,0 +1,253 @@
+.. _gen-ts-cli:
+
+===============================================
+Generate time series data from the command line
+===============================================
+
+This tutorial will show you how to generate some :ref:`experimental time series
+data <gen-ts>` by sampling your `system load`_ using the `CrateDB Shell`_ (aka
+*Crash*) and a little bit of `shell scripting`_.
+
+.. SEEALSO::
+
+    :ref:`gen-ts`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Prerequisites
+=============
+
+CrateDB must be :ref:`installed and running <install-run>`.
+
+Crash is available as `Pip`_ package. `Install`_ it like this:
+
+.. code-block:: console
+
+    $ pip install crash
+
+We designed the commands in this tutorial to be run directly from the `command
+line`_ so that you can experiment with them as you see fit.
+
+.. NOTE::
+
+    This tutorial should work in most POSIX-compatible environments (e.g.,
+    Linux, macOS, and Windows Cygwin). Please let us know if you run into
+    issues.
+
+
+Process ``uptime`` to get your system load
+==========================================
+
+The ``uptime`` command prints some useful information:
+
+.. code-block:: console
+
+    $ uptime
+    16:58  up 3 days,  5:45, 4 users, load averages: 3.15 4.51 5.06
+
+But we don't need all of it.
+
+Let's `pipe`_ the output through `sed`_, and use a `regular expression`_ to
+chop off the bit we don't want (i.e., everything up to and including the space
+before the first load average), like so:
+
+.. code-block:: console
+
+    $ uptime | sed 's/.*load averages: //'
+    3.06 4.46 5.04
+
+This command gives us three load values, corresponding to the one minute
+average, five-minute average, and 15-minute average.
+
+We're always going to want to do this with the output of ``uptime``, so let's
+make things easier for ourselves by defining a `shell function`_:
+
+.. code-block:: console
+
+    $ loads () { uptime | sed 's/.*load averages: //'; }
+
+Now, whenever you want the load averages, you can just run ``loads``:
+
+.. code-block:: console
+
+    $ loads
+    3.06 4.46 5.04
+
+Next, we want to be able to pick individual load averages. There are multiple
+ways to accomplish this. Here's one of them:
+
+.. code-block:: console
+
+    $ echo '3.06 4.46 5.04' | cut -d ' ' -f1
+    3.06
+
+Here, we told `cut`_ to print the value in the first column, which in our case
+is ``3.06``. You could print the second column by changing the ``1`` to ``2``,
+and so on.
+
+So, let's put it all together:
```

Comment:
note to self: both functions can be done in one. no need to seperate

Reply:
nah, I'll keep it like it is. now that the `loads` function has the job of doing cross-platform compat, I think it's a logical separation